cypress/e2e/featureFlags.cy.ts

```diff
@@ -80,7 +80,7 @@ describe('Feature Flags', () => {
             .should('have.value', name + '-updated')
         cy.get('[data-attr=rollout-percentage]').type('{selectall}50').should('have.value', '50')
         cy.get('[data-attr=save-feature-flag]').first().click()
-        cy.wait(100)
```

Comment:
Swapping out all of our `wait`s which can unnecessarily slowing down these tests